recipe/meta.yaml

```diff
@@ -88,65 +86,30 @@ outputs:
         - pip
       run:
         - {{ pin_subpackage('libxgboost', exact=True) }}
-        - {{ pin_subpackage('_py-xgboost-mutex', exact=True) }}
         - python
         - numpy
         - scipy
         - scikit-learn
-        - __cuda  # [cuda_compiler != "None"]
-    test:
-      script: test-py-xgboost.py
-      imports:
-        - xgboost
-
-  - name: py-xgboost-cpu
-    build:
-      skip: true  # [cuda_compiler != "None"]
-    requirements:
-      host:
-        - python
-        - {{ pin_subpackage('py-xgboost', exact=True) }}
-      run:
-        - python
-        - {{ pin_subpackage('py-xgboost', exact=True) }}
-
-  - name: py-xgboost-gpu
-    build:
-      skip: true  # [cuda_compiler == "None"]
-    requirements:
-      host:
-        - python
-        - {{ pin_subpackage('py-xgboost', exact=True) }}
-      run:
-        - python
-        - {{ pin_subpackage('py-xgboost', exact=True) }}
-        - __cuda  # [cuda_compiler != "None"]
-
+      test:
+        script: test-py-xgboost.py
+        imports:
+          - xgboost
 
   - name: xgboost
     build:
-      string: rapidsai_cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler != "None"]
-      string: rapidsai_cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler == "None"]
+      string: rapidsai_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
```

Comment:
So we want to drop the CUDA version from the build string? We are including CUDA version in the build string for other RAPIDS packages. I know the package hash makes this otherwise unique for each CUDA version. Just seems like nice information to have, like the Python version.

Reply:
Yes for 2 reasons:

1. Only one of these packages actually has any dependency on CUDA (`libxgboost`). The rest actually are independent of CUDA version.
2. Since we are using these for CPU & GPU builds (and thinking about upstreaming these changes), including `cuda` in the `build/string` may be confusing for users expecting a CPU package

Reply:
I suppose. It’s just harder to tell if the package was built for CUDA 11 or 12.